src/NUnitFramework/TestBuilder.cs

```diff
@@ -34,7 +34,7 @@
 using NUnit.Framework.Internal.Commands;
 using NUnit.Framework.Internal.Execution;
 
-#if PORTABLE && !NETSTANDARD1_6 && !NETCOREAPP1_0
+#if NETSTANDARD1_3 && !NETSTANDARD1_6 && !NETCOREAPP1_0
```

Comment:
More a question - why do we have `NETCOREAPP1_0` precompilation symbols as well as NETSTANDARD?